rs/bitcoin/kyt/src/providers.rs

```diff
@@ -152,3 +185,31 @@ fn param_transform() -> Option<TransformContext> {
         context: vec![],
     })
 }
+
+pub(crate) fn parse_authorization_header_from_url(
```

Comment:
Do we even need that? Can't we just say regtest with basicauth is not supported? Or is that a somewhat standard setup?

Reply:
That's a good question. I think it is somewhat of a standard setup, so I think it makes sense to have it.

Reply:
Yes, I think it is a somewhat standard setup. I had it, and our system tests also had it.